### DISCUSSION TITLE
Custom branch and bound implementation using HiGHS (rust crate)

### DISCUSSION THREAD

**Discussion by DaveLanday** [ERGO-Code/HiGHS#1738] [Q&A]
I want to use highs in my own implementation of Branch and Bound. I am using the highs.rs crate.  Will I have to set the option mip_max_nodes to 1 in order to achieve this?  If no one has familiarity with the highs.rs crate, how is this done in general? I asked in a discussion at the highs.rs repo but did not get a clarifying or direct response.

**Comment by jajhall** [comment]
If you're wanting to use HiGHS to solve LP problems for a model with integrality values set to HighsVarType::kInteger, just set the solve_relaxation option to true

**Reply by DaveLanday** [reply]
Ok! Thank you; I thought there might be an option to do this but wasn't sure. I will try this!

**Reply by DaveLanday** [reply]
Hi @jajhall, is there a way to read an MPS file using the highs.rs crate, or is an mps file reader missing from the rust implementation? So far, I have only found two mps readers/parsers written in rust and neither can handle lines that look like the following:
`INT1      'MARKER'                 'INTORG'`
